src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java

```diff
@@ -75,7 +75,7 @@ public ArrayList<InsurancePlan> getInsurancePlans() {
      *                                the client's list of insurance plans.
      */
     public InsurancePlan getInsurancePlan(int insuranceId) throws InsurancePlanException {
-        for (InsurancePlan plan : insurancePlans) {
+        for (InsurancePlan plan : this.insurancePlans) {
```

Comment:
Did this cause any bugs? Would we need to make the same change for the other methods in InsurancePlansManager as well?

Reply:
no i was just being specific only, this shouldnt be an issue normally unless theres another object named the same within that local scope.

Reply:
Rights, okie noted!